llmd_dashboard.py

```
@@ -2368,6 +2851,1113 @@ def _winner(v1_val, v2_val, higher_is_better, metric_label):
+        dots = _llmd_health_dots_html(h)
+        n_losses_h = len(data["loss_combos"])
+        st.markdown(
+            f"""<div class="overview-card"><details><summary>
+<div class="overview-card-title">Release Health Score</div>
+<div class="overview-card-value {h_cls}">
+{h} {dots}
+</div>
+</summary>
+<div class="overview-card-detail">
+Non-regression rate: {data["win_rate"]:.0f} %
+= ({data["total_cmp"]} − {n_losses_h} losses) / {data["total_cmp"]} total.<br>
```

Ruff (0.15.18)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)